projects/planner/src/app/gas-blender/gas-blender.component.html

```diff
@@ -130,7 +131,7 @@
                         <label for="o2UnitPrice" class="form-label">Unit price O2:</label>
                         <input formControlName="o2UnitPrice" class="form-control" type="number"
                                (input)="applyChange()" [min]="ranges.money[0]" [max]="ranges.money[1]" step="1"
-                               required [class.is-invalid]="o2UnitPriceInvalid" id="o2UnitPrice" />
+                               required [class.is-invalid]="o2UnitPriceInvalid"/>
```

Comment:
How do you imput the value, if you removed the Id element.

Reply:
I tried to find o2UnitPrice elswhere - trough final calculated O2 price and it worked, but it is wrong